src/main/java/com/faforever/client/login/LoginController.java

```diff
@@ -95,7 +103,7 @@ private void onLoginFailed(Throwable e) {
     logger.warn("Login failed", e);
     Platform.runLater(() -> {
       if (!(e instanceof CancellationException)) {
-        loginErrorLabel.setText(e.getCause().getLocalizedMessage());
+        loginErrorLabel.setText(e.getCause().getLocalizedMessage().toString().replace(". ",".\n"));
```

Comment:
Can you help me fixing wrapping issue wraptext is already "true". What is the matter then? maxHeigth?